docs/reference/commandline/stack_deploy.md

```diff
@@ -38,8 +38,11 @@ Options:
 
 ## Description
 
-Create and update a stack from a `compose` or a `dab` file on the swarm. This command
-has to be run targeting a manager node.
+Create and update a stack from a `compose` or a `dab` file on the swarm.
+
+> **Note**: This is a cluster management command, and must be executed on a swarm
```

Comment:
Wasn't sure how to deal with k8s as an orchestrator here; this command can be run both on Swarm or Kubernetes, so we may need a different description to either make it generic enough to fit both use-cases, or extend the description to explicitly call out both use-cases

Reply:
Maybe add a something like: ```**Note**:If you use swarm as an orchestrator, this is a cluster management command...``` ?